app/views/hyrax/embargoes/_embargo_history.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% curation_concern.embargo_history.each do |entry| %>
+    <% unless entry.empty? %>
```

Comment:
Prefer `if entry.present?` (I also don't believe that `nil` responds to `empty?`)